server/templates/staff/course/course.edit.html

```diff
@@ -33,6 +33,7 @@ <h3 class="box-title">Edit {{ current_course.display_name_with_semester }}</h3>
                     {{ forms.render_field(form.institution, label_visible=true, placeholder='UC Berkeley', type='text') }}
                     {{ forms.render_field(form.website, label_visible=true, placeholder='http://cs61a.org/', type='text') }}
                     <!-- {{ forms.render_field(form.timezone, label_visible=true, type='select') }} -->
+                    {{ forms.render_field(form.autograder_url, label_visible=true, placeholder='http://autograder.cs61a.org/', type='text') }}
```

Comment:
This should be `https://autograder.cs61a.org`. Rather than just using `placeholder`, can you please set the value as `https://autograder.cs61a.org` if a custom URL is not configured.